airbyte-workers/src/main/java/io/airbyte/workers/temporal/CheckConnectionWorkflow.java

```diff
@@ -92,15 +92,20 @@ public StandardCheckConnectionOutput run(JobRunConfig jobRunConfig,
                                              IntegrationLauncherConfig launcherConfig,
                                              StandardCheckConnectionInput connectionConfiguration)
         throws TemporalJobException {
-      return new TemporalAttemptExecution<>(workspaceRoot, jobRunConfig, (jobRoot) -> {
-        final IntegrationLauncher integrationLauncher = new AirbyteIntegrationLauncher(
-            launcherConfig.getJobId(),
-            Math.toIntExact(launcherConfig.getAttemptId()),
-            launcherConfig.getDockerImage(),
-            pbf);
-
-        return new DefaultCheckConnectionWorker(integrationLauncher).run(connectionConfiguration, jobRoot);
-      }).get();
+      return new TemporalAttemptExecution<>(
```

Comment:
This method is short enough it's digestable as is. Although it'll still be nice to have type information and separate the constructor call from the `.get` call. Not blocking.